config-example.yaml

```diff
@@ -58,6 +58,10 @@ handlers:
     # Transactions with lower amount mojos will be ignored
     # Use this to avoid notification spam during dust storms
     min_mojos_amount:  5 # default: 0
+  wallet_peak_handler:
+    # Observed delay is usually about a minute.
+    # If the delay is consistently over this for the WALLET notify_threshold_seconds value, the keep-alive triggers.
+    max_drift_seconds: 120
```

Comment:
Considering wallets are very much nice-to-have for a farming operation, I’d push for a more lax limits here. Maybe 5 minutes? Reading the PR description, 2 minutes sounds like its too close to comfort when accounting for raspberry and dust storms.

Reply:
Makes sense, upped it to 5 minutes. Anyone that actually cares about wallet latency can go tweak it themselves.